build/package/LICENSE

```diff
@@ -319,69 +236,108 @@ conditions of the following licenses.
     OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
     THE SOFTWARE.
 
-- 'os-paths' extension/dist/ext/extension.js
-  This product bundles 'os-paths' from the above files.
-  These files are available under the MIT License:
-
+- 'ansi-regex' in extension/dist/ext/extension.js
+- 'ansi-styles' in extension/dist/ext/extension.js
+- 'is-fullwidth-code-point' in extension/dist/ext/extension.js
+- 'string-width' in extension/dist/ext/extension.js
+- 'wrap-ansi' in extension/dist/ext/extension.js
+  This product bundles 'ansi-regex', 'ansi-styles', 'is-fullwidth-code-point', 'string-width' and 'wrap-ansi' from the above files.
+  These packages are available under the MIT License:
     MIT License
 
-    Copyright (c) Roy Ivy III <rivy.dev@gmail.com>
-    Copyright (c) Sindre Sorhus <sindresorhus@gmail.com> (sindresorhus.com)
+    Copyright (c) Sindre Sorhus <sindresorhus@gmail.com> (https://sindresorhus.com)
+
+    Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+- 'arg' in extension/dist/ext/extension.js
+	This product bundles 'arg' from the above files.
```

Comment:
It's not obvious in this diff, but these files mix tabs and spaces in some places. It might be worth converting them to spaces just so we're consistent.